### DISCUSSION TITLE
Pass extra param for passwordless login url generation

### DISCUSSION THREAD

**Discussion by Antoine-O** [meteor/meteor#13413] [Feature Requests]
Hello,

actually the code is

```
Accounts.sendLoginTokenEmail = ({ userId, sequence, email, extra = {} }) => {
        const user = getUserById(userId);
        const url = Accounts.urls.loginToken(email, sequence);
```

could you please add extra var to the logintoken method  like this ?

```
Accounts.sendLoginTokenEmail = ({ userId, sequence, email, extra = {} }) => {
        const user = getUserById(userId);
        const url = Accounts.urls.loginToken(email, sequence,extra);
```

thanks

**Comment by StorytellerCZ** [comment]
This doesn't look like a feature request, but instead looks to me like a bug or unfinished feature. Given the ease of the fix you could have just submitted a PR.

**Comment by nachocodoner** [comment]
This feature is included in the 3.1.1 release. To update, use `meteor update --release 3.1.1`.
